.github/workflows/osx.yml

```diff
@@ -29,15 +29,14 @@ jobs:
           ref: "1.8.32"
           path: "aws-sdk-cpp"
 
-      # - name: Patch aws-sdk-cpp CMakeLists
-      #   run: |
-      #     cd aws-sdk-cpp
-      #     awk 'NR==6 {printf "if(POLICY CMP0114)\n    cmake_policy(SET CMP0114 OLD)\nendif()\n"} 1' third-party/CMakeLists.txt > CMakeLists_patched.txt
-      #     mv CMakeLists_patched.txt third-party/CMakeLists.txt
-
-      - name: Switch Xcode version
+      - name: Install CMake v3.18.5
         run: |
-          sudo xcode-select -s "/Applications/Xcode_12.app"
+          brew uninstall cmake
+          curl -L https://github.com/Kitware/CMake/releases/download/v3.18.5/cmake-3.18.5.tar.gz | tar xz
```

Comment:
This looks good so far. We should add a comment noting *why* we locked this down and cutting an issue to revisit this in a month or two.

Reply:
Comment added. Unfortunately brew doesn't allow to install older versions of CMake.